pkgs/tools/misc/ollama-webui/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildNpmPackage
+, nodePackages
+, fetchFromGitHub
+, runtimeShell
+}:
+
+# We just package the JS frontend part, not the Python reverse-proxy backend.
+# NixOS can provide any another reverse proxy such as nginx.
+buildNpmPackage rec {
+  pname = "ollama-webui";
+  # ollama-webui doesn't tag versions yet.
+  version = "0.0.0-unstable-2023-12-22";
+
+  src = fetchFromGitHub {
+    owner = "ollama-webui";
+    repo = "ollama-webui";
+    rev = "77c1a77fccb04337ff95440030cd051fd16c2cd8";
+    hash = "sha256-u7h2tpHgtQwYXornslY3CZjKjigqBK2mHmaiK1EoEgk=";
+  };
+  # dependencies are downloaded into a separate node_modules Nix package
+  npmDepsHash = "sha256-SI2dPn1SwbGwl8093VBtcDsA2eHSxr3UUC+ta68w2t8=";
+
+  # We have to bake in the default URL it will use for ollama webserver here,
+  # but it can be overriden in the UI later.
+  PUBLIC_API_BASE_URL = "http://localhost:11434/api";
+
+  # The path '/ollama/api' will be redirected to the specified backend URL
+  OLLAMA_API_BASE_URL = PUBLIC_API_BASE_URL;
```

Comment:
the env var only seems to be set after the build has succeeded
https://github.com/ollama-webui/ollama-webui/blob/main/Dockerfile#L22
does the build fail without ?
if not, this is probably something that should be set in the service, not in the package.